### ISSUE TITLE
Focus indicator is potentially confusing for tabs and accordion components

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#1405]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: Usability feedback only, there is no WCAG related guidelines.
**Issue ID**: DAC_Issue21
**URLs**: 
- http://govuk-frontend-v3.herokuapp.com/full-page-examples/bank-holidays#tab_england-and-wales
- http://govuk-frontend-v3.herokuapp.com/full-page-examples/service-manual-topic

## Screen Shots
![](https://user-images.githubusercontent.com/2445413/58713289-276b2600-83ba-11e9-8835-088176ad445e.png)
![](https://user-images.githubusercontent.com/2445413/58713294-29cd8000-83ba-11e9-9dd1-35a3034404c9.png)

The focus indicator did not surround elements as expected and only appeared at the top edge of the tab. This was confusing to keyboard only users in some cases as it was not clear which element was in focus.

## Current Code Ref(s)
```html
.js-enabled .govuk-tabs__tab--selected:focus, .js-enabled .govuk-tabs__tab--selected.\:focus { box-shadow: inset 0 4px 0 0 #ffdd00, inset 0 8px 0 0 #0b0c0c;
```

## Keyboard only comments
> “The highlighting being used on the tabs consists of a line appearing above it. A keyboard only user would expect highlighting to appear around the whole element.”

> “After the ‘click to expand link’ was selected, focus went to the line above it. At this point it is not obvious when the line was selected and that it will close the ‘click to expand’ element. As it appeared at the top of the link, I was not sure which link it was highlighting.”

## Solution
It is recommended that the entire area receives a visual focus indication, to ensure that it is clear which element is active.

**Comment by nickcolley** [comment]
@dashouse has considered this feedback and [created an example that would address these concerns](https://github.com/alphagov/govuk-frontend/pull/1394), he is going to explore this further and based on that we'll make a decision on feedback.

**Comment by nickcolley** [comment]
Dave has explored this in https://github.com/alphagov/govuk-frontend/pull/1394 and we have decided as a team to go ahead with this simplification which will resolve this issue.

**Comment by aliuk2012** [comment]
Fixed in #1424
